src/desktop/analytics/account_login.js

```diff
@@ -7,6 +7,17 @@ var trackViewLogin = function() {
   analytics.track('Viewed login form')
 }
 
+const trackLogin = options => {
+  analytics.track('Successfully logged in', options)
```

Comment:
I added a separate tracking event for login here.

Reply:
Before we were tracking successful login event via the `auth:login` mediator. It was located within the `account_creation.js` file which was a little confusing. I'm thinking we can deprecate `auth:login` once we flip the switch for the new auth.